src/field_specials.c

```diff
@@ -368,17 +347,15 @@ bool32 ShouldDoWallyCall(void)
     {
         switch (gMapHeader.mapType)
         {
-            case MAP_TYPE_TOWN:
-            case MAP_TYPE_CITY:
-            case MAP_TYPE_ROUTE:
-            case MAP_TYPE_OCEAN_ROUTE:
-                if (++(*GetVarPointer(VAR_WALLY_CALL_STEP_COUNTER)) < 250)
-                {
-                    return FALSE;
-                }
-                break;
-            default:
+        case MAP_TYPE_TOWN:
+        case MAP_TYPE_CITY:
+        case MAP_TYPE_ROUTE:
+        case MAP_TYPE_OCEAN_ROUTE:
+            if (++(*GetVarPointer(VAR_WALLY_CALL_STEP_COUNTER)) < 250)
                 return FALSE;
```

Comment:
There are no parenthesis around this like in the other functions. Either they were forgotten or they shouldn't be in the other functions.

Reply:
The parenthesis format here is consistent with all the other `if (++(*GetVarPointer(VAR)) < NUM)` in this file. In any case this PR has been merged already, if you'd like to propose a change please open a PR or discuss it in the Discord